.gitignore

```diff
@@ -2,12 +2,12 @@
 *.log
 *.yaml
 
-.gitignore
 lib64
 pip-selfcheck.json
 pywikibot.lwp
 throttle.ctrl
 user-password.py
+user-config.py
 pyvenv.cfg
 *.yaml
```

Comment:
Remove this duplicated line while you're at it.

Reply:
To clarify, I meant line the line with `*.yaml` (which is also a bit further up), which isn't obvious the way Github displays it.

Reply:
> To clarify, I meant line the line with `*.yaml` (which is also a bit further up), which isn't obvious the way Github displays it.

Do we need to add a local config file for flake8?